CWP/papers/roadmap/authors/address.txt

```diff
@@ -0,0 +1,112 @@
+ANL: High Energy Physics Division, Argonne National Laboratory, Argonne IL, USA
+BNL: Physics Department, Brookhaven National Laboratory, Upton NY, USA
+Caltech: California Institute of Technology, Pasadena, California, USA
+CBPF: Centro Brasileiro de Pesquisas Fisicas, Rio de Janeiro, Brazil
+CCIN2P3: Centre de Calcul de l’IN2P3, Villeurbanne, Lyon, France
+CERN: CERN, Geneva, Switzerland
+CIEMAT: Centro de Investigaciones Energéticas Medioambientales y Tecnológicas (CIEMAT), Madrid, Spain
+CHICAGO: Enrico Fermi Institute, University of Chicago, Chicago IL, USA
+CINVESTAV: Cinvestav, Mexico City, Mexico
+CNAF: Centro Nazionale Analisi Fotogrammi (CNAF), INFN, Bologna, Italy
+Cornell: Cornell University, Ithaca, USA
+CPHC: Center for High Performance Computing, Cape Town, South Africa
+DESY: Deutsches Elektronen-Synchrotron, Hamburg, Germany
+EPFL: Institute of Physics, Ecole Polytechnique Federale de Lausanne (EPFL), Lausanne, Switzerland
```

Comment:
École Polytechnique Fédérale de Lausanne

Reply:
All these things should be fixed in the Google Docs, IMO. I'll check and update the PR then.